oss-internship-2020/gdal/raster.cc

```diff
@@ -0,0 +1,144 @@
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+#include <syscall.h>
+
+#include <iostream>
+
+#include "gdal_sapi.sapi.h"
```

Comment:
For various internal reasons, please add a `// NOLINT(build/include)` at the end of this line.